### PR TITLE
LW-10576 Set not zero value for pool and stake key deposit in local-network

### PR DESCRIPTION
**Pull request by iccicci**
# Context

In our `local-network` we have the dangerous setting of `0` for stake key deposit and stake pool deposit.
This makes us less confident about our tests.

# Proposed Solution

Changed the two settings as per `mainnet`.

### CODE REVIEW

**Review by AngelCastilloB — DISMISSED** [review]
Awesome job @iccicci 🍰

File: packages/e2e/local-network/scripts/pools/update-node-utils.sh [hunk 1]
```diff
@@ -129,7 +129,7 @@ updatePool() {
     --protocol-params-file ${SP_NODE_ID}/params.json | awk '{ print $1 }')
 
   initialBalance=$(getAddressBalance "$genesisAddr")
-  txOut=$((initialBalance - fee))
+  txOut=$((initialBalance - fee - 2000000))
```
Comment (mkazlauskas) [inline packages/e2e/local-network/scripts/pools/update-node-utils.sh:132]: Is this for keyDeposit that was previously 0?
```suggestion
  txOut=$((initialBalance - fee - keyDeposit))
```

File: packages/cardano-services/test/StakePool/StakePoolHttpService.test.ts [hunk 2]
```diff
@@ -964,7 +964,7 @@ describe('StakePoolHttpService', () => {
             });
             it('pledgeMet false, status activating, and condition', async () => {
```
Comment (mkazlauskas) [inline packages/cardano-services/test/StakePool/StakePoolHttpService.test.ts:965]: Not sure the original intent of the test, but that's what it's doing now
```suggestion
            it('pledgeMet false, status active, and condition', async () => {
```

**Review by mkazlauskas — CHANGES_REQUESTED** [review]

File: packages/e2e/local-network/scripts/pools/update-node-utils.sh (hunk 1 above)
Comment (iccicci) [reply packages/e2e/local-network/scripts/pools/update-node-utils.sh]: Yes. 001e4043df should solve this

File: packages/cardano-services/test/StakePool/StakePoolHttpService.test.ts (hunk 2 above)
Comment (iccicci) [reply packages/cardano-services/test/StakePool/StakePoolHttpService.test.ts]: It's hard to say which was the original intent of the test... it's inside a `identifier & status & pledgeMet filters` describe block, so I guess the intent of the test is to check that logical expression between filters work as expected.
Anyway the status of a stake pool in the fixtures is strongly dependent on the time when the snapshot was taken... probably recent changes introduced make no pools to have the right combination to match the filter in this tests so I had only two options: disabling the test or change it as I did.
In my opinion, tests about stake pool status should be done only as long running tests which prepare the pools as per their needs, so we can be sure about the time when they run.
Until that improvement is done... 94e3d9438a should solve this

**Review by mkazlauskas — APPROVED** [review]
Nice work! :rocket:

**Review by AngelCastilloB — APPROVED** [review]

**Pull request merged**